Tests/JWEDirectEncryptionTests.swift

```diff
@@ -43,7 +43,8 @@ class JWEDirectEncryptionTests: CryptoTestCase {
         121,  25,   4,  85, 220, 144, 215, 110,
         130,  17,  68, 228, 129, 138,   7, 130
     ])
-    
+
+    @available(*, deprecated)
```

Comment:
Hmm, what's the plan for all those deprecated tests? Do you write new ones?

Reply:
No, they use a deprecated method (the decrypt method; which should be fine for the test since we're not attacking ourselves 😉). If I don't mark them deprecated we get a warning for each decrypt call. If we want to get rid of the annotations, we just have to use the other decrypt method (the one which takes a verifier).

I can update that now if you want?

Reply:
That's what I meant. But it's fine for me.